oxidation/libparsec/crates/crypto/Cargo.toml

```diff
@@ -18,14 +18,16 @@ libsodium-sys = { version = "0.2.7", optional = true }
 
 argon2 = { version = "0.4.1", optional = true }
 blake2 = { version = "0.10.0", optional = true }
-crypto_box = { version = "0.7.0", optional = true, features = ["std"] }
+crypto_box = { version = "0.8.2", optional = true, features = ["std"] }
 digest = { version = "0.10.5", optional = true }
-ed25519-dalek = { version = "1.0.1", optional = true }
+ed25519-dalek = { version = "1.0.1" }
 generic-array = { version = "0.14.6", optional = true, features = ["serde"] }
 lazy_static = { version = "1.4.0", optional = true }
-sha2 = { version = "0.10.6", optional = true }
-x25519-dalek = { version = "1.2.0", optional = true }
-xsalsa20poly1305 = { version = "0.8.0", optional = true }
+# sodiumoxide don't provide rsa key
+rsa = { version = "0.7.2" }
+sha2 = { version = "0.10.6" }
+x25519-dalek = { version = "2.0.0-pre.1", optional = true }
```

Comment:
Pre-stable for a crypto library is a no go

Reply:
Rip :(